apps/site/pages/components/organisms/product-grid.mdx

```diff
@@ -0,0 +1,203 @@
+
+It's a section generally used on PLP pages to list the products available in the store.
+
+</header>
+
+The `ProductGrid` is a compound of the following:
+
+- `ProductGrid`: wraps the product grid items.
+- `ProductGridItem`: wraps the product inside a list item.
+
+\*We recommend using the [ProductCard](/components/molecules/product-card) as the direct child of the ProductGridItem.
+
+---
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { ProductGrid, ProductGridItem } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/organisms/ProductGrid/styles.scss'
+```
+
+---
+
+## Usage
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection>
+      <ProductGrid>
+           {Array.from({ length: 5 }, (_, index) => (
+             <ProductGridItem>
+               <ProductCard>
+                 <ProductCardImage>
+                   <img
+                     data-fs-image
+                     src={product.image[0].url}
+                     alt={product.image[0].alternateName}
+                   />
+                 </ProductCardImage>
+                 <ProductCardContent
+                   title={product.isVariantOf.name}
+                   price={{
+                     value: product.offers.offers[0].price,
+                     listPrice: product.offers.offers[0].listPrice,
+                     formatter: useFormattedPrice,
+                   }}
+                 />
+               </ProductCard>
+             </ProductGridItem>
+           ))}
+       </ProductGrid>
+
+    </OverviewSection>
+
+  </Tab>
+  <Tab>
+
+    ```tsx
+
+<ProductGridItem>
+  <ProductCard>
+    <ProductCardImage>
+      <img
+        data-fs-image
+        src={product.image[0].url}
+        alt={product.image[0].alternateName}
+      />
+    </ProductCardImage>
+    <ProductCardContent
+      title={product.isVariantOf.name}
+      price={{
+        value: product.offers.offers[0].price,
+        listPrice: product.offers.offers[0].listPrice,
+        formatter: useFormattedPrice,
+      }}
+    />
+  </ProductCard>
+</ProductGridItem>
+```
+
+  </Tab>
+</Tabs>
+
+---
+
+## Props
+
+### ProductGrid
+
+<ProductGridPropsSection component="ProductGrid" />
+
+### ProductGridItem
+
+<ProductGridPropsSection component="ProductGridItem" />
+
+---
+
+## Design Tokens
+
+<TokenTable>
+  <TokenRow token="--fs-product-grid-gap-mobile" value="var(--fs-grid-gap-0)" />
+  <TokenRow
+    token="--fs-product-grid-gap-tablet"
+    value="var(--fs-product-grid-gap-mobile)"
+  />
+  <TokenRow
+    token="--fs-product-grid-gap-desktop"
+    value="var(--fs-grid-gap-2)"
+  />
+  <TokenDivider />
+  <TokenRow token="--fs-product-grid-columns-mobile" value="2" />
+  <TokenRow token="--fs-product-grid-columns-tablet" value="4" />
+  <TokenRow
+    token="--fs-product-grid-columns-desktop"
+    value="var(--fs-product-grid-columns-tablet)"
+  />
+</TokenTable>
+
+---
```

Comment:
What do you think of adding the same `Variants` section from [ProductShelf](https://faststore-site-git-feat-product-shelf-fs-777-faststore.vercel.app/components/organisms/product-shelf#variants)?

Reply:
Hm, I don't think they are very related. But I think we can add `ProductShelf` as Related component of `ProductGrid`.